app/tasks/doctor.php

```diff
@@ -25,36 +25,36 @@
         $domain = new Domain(App::getEnv('_APP_DOMAIN'));
 
         if(!$domain->isKnown() || $domain->isTest()) {
-            Console::log('🔴 Hostname has a public suffix');
+            Console::log('🔴 Hostname has a public suffix ('.$domain->get().')');
         }
         else {
-            Console::log('🟢 Hostname has a public suffix');
+            Console::log('🟢 Hostname has a public suffix ('.$domain->get().')');
         }
         
         $domain = new Domain(App::getEnv('_APP_DOMAIN_TARGET'));
 
         if(!$domain->isKnown() || $domain->isTest()) {
-            Console::log('🔴 CNAME target has a public suffix');
+            Console::log('🔴 CNAME target has a public suffix ('.$domain->get().')');
         }
         else {
-            Console::log('🟢 CNAME target has a public suffix');
+            Console::log('🟢 CNAME target has a public suffix ('.$domain->get().')');
         }
         
-        if(App::getEnv('_APP_OPENSSL_KEY_V1', 'your-secret-key') === 'your-secret-key') {
+        if(App::getEnv('_APP_OPENSSL_KEY_V1', 'your-secret-key') === 'your-secret-key' || empty(App::getEnv('_APP_OPENSSL_KEY_V1', 'your-secret-key'))) {
```

Comment:
@eldadfux I think the second part is useless, as 'your-secret-key' is used as default, if environment variable is empty. (`empty == "" == null == 0 == []`)

Reply:
Maybe we should change the message description, because using an empty key is not way better than using the default one.

Reply:
I think you can't enter an empty key, as if so, it would be replaced by default variable in the second param of `App::getEnv`.

Reply:
Or you meant the second part to be `empty(App::getEnv('_APP_OPENSSL_KEY_V1'))` only, and change the text accordingly, or even make it its own case to explicitely stating that key is empty.

Reply:
Yes, that was my mistake, I'll fix it :)